invenio_rdm_records/services/schemas/record.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
```

Comment:
this is only refactoring since previously the schema was inside `__init__.py` - not easy to find